packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -1775,6 +1775,7 @@ export type CustomAvatarOptions = {
     size?: PersonaSize;
     styles?: IStyleFunctionOrObject<IPersonaStyleProps, IPersonaStyles>;
     text?: string;
+    isActive?: boolean;
```

Comment:
should this be `isRaisedHandActive`?

Reply:
I would say change this to isActive. In  this context of this component it simply adds a border, that could be used for anything

Reply:
This is part of CustomAvatar and it is a components

Reply:
We don't apply it internally to these react-components correct? If we are passing in a property into the component, only to pass it on to a callback (the callback which is also passed into the component) I don't see the value in adding this to the component interface

Reply:
I agree with James, we probably don't need to add these props. See my comments [here](https://github.com/Azure/communication-ui-library/pull/3793/files#r1401284634)